### PR TITLE
Switch to bcr based SEAL

### PR DESCRIPTION
**Pull request by anakinxc**

### CODE REVIEW

File: MODULE.bazel [hunk 1]
```diff
@@ -59,7 +60,6 @@ use_repo(
     "curve25519-donna",
     "kuku",
     "perfetto",
-    "seal",
```
Comment (tongke6) [inline MODULE.bazel:62]: please drop seal in repository too

**Review by tongke6 — APPROVED** [review]
LGTM

File: MODULE.bazel (hunk 1 above)
Comment (anakinxc) [reply MODULE.bazel:62]: Done

**Review by tongke6 — DISMISSED** [review]
LGTM

**Review by tongke6 — APPROVED** [review]
LGTM

**Review by rivertalk — APPROVED** [review]

**Pull request merged**